src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Colors {
+    Black, White, Green, Blue, Red, Purple, Yellow
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-forget-how-to-name-constants-according-to-style-guide-the-same-with-naming-of-enum-values